recipes/webob/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: webob
+  version: "1.6.2"
+
+source:
+  fn: WebOb-1.6.2.tar.gz
+  url: https://pypi.python.org/packages/df/40/009a807e85593db8f21ec4d27807a83b6435eed9fd7ef3764d864479b930/WebOb-1.6.2.tar.gz
+  md5: cc88bf68d6bf879c2d2145c86b6f5010
+
+build:
+  number: 0
+  script: python setup.py install 
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
If the package uses setuptools you should add the extra args in the script call on line 12.